src/frontend/planner_test/tests/testdata/nexmark_watermark.yaml

```diff
@@ -0,0 +1,2157 @@
+# This file is automatically generated. See `src/frontend/planner_test/README.md` for more information.
+- id: create_sources
+  sql: "CREATE SOURCE nexmark (\n  event_type BIGINT,\n  person STRUCT<\"id\" BIGINT,\n                \"name\" VARCHAR,\n                \"email_address\" VARCHAR,\n                \"credit_card\" VARCHAR,\n                \"city\" VARCHAR,\n                \"state\" VARCHAR,\n                \"date_time\" TIMESTAMP,\n                \"extra\" VARCHAR>,\n  auction STRUCT<\"id\" BIGINT,\n                 \"item_name\" VARCHAR,\n                 \"description\" VARCHAR,\n                 \"initial_bid\" BIGINT,\n                 \"reserve\" BIGINT,\n                 \"date_time\" TIMESTAMP,\n                 \"expires\" TIMESTAMP,\n                 \"seller\" BIGINT,\n                 \"category\" BIGINT,\n                 \"extra\" VARCHAR>,\n  bid STRUCT<\"auction\" BIGINT,\n             \"bidder\" BIGINT,\n             \"price\" BIGINT,\n             \"channel\" VARCHAR,\n             \"url\" VARCHAR,\n             \"date_time\" TIMESTAMP,\n             \"extra\" VARCHAR>,\n  date_time TIMESTAMP AS\n    CASE\n        WHEN event_type = 0 THEN (person).date_time\n        WHEN event_type = 1 THEN (auction).date_time\n        ELSE (bid).date_time\n    END,\n  WATERMARK FOR date_time AS date_time - INTERVAL '4' SECOND\n) WITH (\n    connector = 'nexmark',\n    nexmark.split.num = '2',\n    nexmark.min.event.gap.in.ns = '100'\n);\n\nCREATE VIEW PERSON as \nselect \n  (person).id, (person).name, (person).email_address, (person).credit_card, (person).city, (person).state, \n  date_time, (person).extra\nfrom \n  nexmark \nwhere event_type = 0;\n\nCREATE VIEW AUCTION as \nselect \n  (auction).id, (auction).item_name, (auction).description, (auction).initial_bid, (auction).reserve, date_time,\n  (auction).expires, (auction).seller, (auction).category, (auction).extra\nfrom \n  nexmark \nwhere event_type = 1;\n\nCREATE VIEW BID as \nselect \n  (bid).auction, (bid).bidder, (bid).price, (bid).channel, (bid).url, date_time, (bid).extra\nfrom \n  nexmark \nwhere event_type = 2;\n"
```

Comment:
....
I wrote in the normal way
and the generator compressed them into a single-line string...